packages/fastify/README.md

```diff
@@ -17,23 +17,20 @@ fastify.register(ByuJwtProvider, {
+  }]
 })
 
 await fastify.listen({ port: 3000 }).catch(console.error)
 ```
 
 ## Options
 
-| property              | type    | default     | description                                                                                                                                                                                       |
-|-----------------------|---------|-------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| prefix                | string  | `undefined` | Will only authenticate routes matching this prefix.                                                                                                                                               |
-| development           | boolean | false       | skips JWT verification for development purposes but will throw an error if NODE_ENV is set to `production`.                                                                                       |
-| basePath              | string  | `undefined` | will validate that the audience starts with the provided basePath in production.                                                                                                                  |
-| byuJwtOptions         | object  | `undefined` | an object that contains any ByuJwt options passed in. See the [BYU JWT](https://byu-oit.github.io/byu-jwt-nodejs/modules/BYU_JWT.html#md:options) Documentation for a full list of those options. |
+| property         | type    | default     | description                                                                                                                                                                               |
+|------------------|---------|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| prefix           | string  | `undefined` | Will only authenticate routes matching this prefix.                                                                                                                                       |
+| development      | boolean | false       | skips JWT verification for development purposes but will throw an error if NODE_ENV is set to `production`.                                                                               |
+| basePath         | string  | `undefined` | will validate that the audience starts with the provided basePath in production.                                                                                                          |
+| ...byuJwtOptions | object  | `undefined` | additional ByuJwt options that are passed in. See the [BYU JWT](https://byu-oit.github.io/byu-jwt-nodejs/modules/BYU_JWT.html#md:options) Documentation for a full list of those options. |
```

Comment:
Let's put the description on this line above the table instead of a part of the table.